packages/components/src/ToggleArrow/src/ToggleArrow.module.css

```diff
@@ -0,0 +1,49 @@
+.hop-ToggleArrow  {
+    /* Default */
```

Comment:
isn't all those props just to match the text with which they are associated? why not reduce the complexity of this component and juste using color = inherit. 

This way you wouldn't have a bunch of complex state that means nothing to this component?

Reply:
it was so that the components using this didn't have to define all the colors. plus if the color is ever updated, we only need to change it in one place.